base-helm-configs/glance/glance-helm-overrides.yaml

```diff
@@ -1,5 +1,5 @@
 # radosgw, rbd, swift or pvc
-storage: swift  # Use override to switch to pvc in dev env
+storage: pvc  # Use override to switch to pvc in dev env
```

Comment:
should we change the comment, or maybe remove it?

Reply:
Remove comment